src/components/Datatable/Pagination/NavButton.tsx

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { NavButtonProps } from './Pagination.types';
+import { IconButton } from '../../IconButton';
+
+const NavButton: NavButtonProps = styled(IconButton).attrs(
+  ({ isDisabled, ...props }) => ({
+    ...props,
+    disabled: isDisabled,
+  }),
+)`
+  display: inline-block;
+
+  width: ${pxToRem(48)};
+  height: ${pxToRem(32)};
+  padding: ${pxToRem(8, 16)};
+  margin: ${pxToRem(0, 4)};
+  border-radius: ${pxToRem(3)};
+
+  background-color: ${({ isDisabled }) =>
+    getColor(isDisabled ? 'transparent' : 'graphite5H')};
+  background-color: ${({ isDisabled }) =>
```

Comment:
Why do we have the `background-color` twice here?

Reply:
copy/paste mistake, thanks for noticing